<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplerWidget</class>
 <widget class="QWidget" name="SamplerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sampler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="spatialDataLayerLabel">
       <property name="text">
        <string>Spatial Data Layer:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QgsMapLayerComboBox" name="spatialDataLayerComboBox">
       <property name="allowEmptyLayer">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="samplerTypeLabel">
       <property name="text">
        <string>Sampler Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="samplerTypeComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dtmLayerLabel">
       <property name="text">
        <string>DTM Layer:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QgsMapLayerComboBox" name="dtmLayerComboBox">
       <property name="allowEmptyLayer">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="geologyLayerLabel">
       <property name="text">
        <string>Geology Layer:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsMapLayerComboBox" name="geologyLayerComboBox">
       <property name="allowEmptyLayer">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="decimationLabel">
       <property name="text">
        <string>Decimation:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="decimationSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="spacingLabel">
       <property name="text">
        <string>Spacing:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QgsDoubleSpinBox" name="spacingSpinBox">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
       <property name="value">
        <double>200.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Run Sampler</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
